pkg/controller/networkpolicy/validate.go

```diff
@@ -549,6 +556,31 @@ func numFieldsSetInPeer(peer crdv1alpha1.NetworkPolicyPeer) int {
 	return num
 }
 
+// checkSelectorsLabels validates labels used in all selectors passed in.
+func checkSelectorsLabels(selectors []*metav1.LabelSelector) (string, bool) {
```

Comment:
nit: `func checkSelectorsLabels(selectors ...*metav1.LabelSelector) (string, bool) {` then you don't need to create a slice on caller side: `checkSelectorsLabels(s.PodSelector, s.NamespaceSelector, s.ExternalEntitySelector)`

Reply:
Thanks. Changed.